runtimes/eoapi/stac/eoapi/stac/api.py

```diff
@@ -0,0 +1,201 @@
+"""eoapi.stac.api: custom StacAPI class."""
+
+from typing import Type
+
+import attr
+from stac_fastapi.api import app
+from stac_fastapi.api.models import APIRequest, GeoJSONResponse
+from stac_fastapi.api.routes import create_async_endpoint
+from stac_pydantic import api
+from stac_pydantic.api.collections import Collections
+from stac_pydantic.shared import MimeTypes
+
+from .extensions import HTMLorJSONGetRequest
+
+
+@attr.s
+class StacApi(app.StacApi):
+    """Custom StacAPI."""
+
+    landing_get_model: Type[APIRequest] = attr.ib(default=HTMLorJSONGetRequest)
+    conformance_get_model: Type[APIRequest] = attr.ib(default=HTMLorJSONGetRequest)
+
+    def register_landing_page(self):
+        """Register landing page (GET /).
+
+        Returns:
+            None
+        """
+        self.router.add_api_route(
+            name="Landing Page",
+            path="/",
+            response_model=(
+                api.LandingPage if self.settings.enable_response_models else None
+            ),
+            responses={
+                200: {
+                    "content": {
+                        MimeTypes.json.value: {},
+                        MimeTypes.html.value: {},
```

Comment:
it's a shame but we almost need a custom StacApi class just to add the response content 🤷‍♂️